draco/core/task.py

```diff
@@ -389,8 +393,20 @@ def _save_output(self, output):
             outfile = os.path.expanduser(outfile)
             outfile = os.path.expandvars(outfile)
 
+            if self.output_chunking:
+                try:
+                    output.chunkify()
```

Comment:
As discussed, I'm not sure if we want this one. I'd like to know if it really is necessary first.

Reply:
I ran it without and it's not needed :+1: